src/sentry/static/sentry/app/actionCreators/modal.tsx

```diff
@@ -1,12 +1,24 @@
 import React from 'react';
 import {css} from 'react-emotion';
+import {ModalHeader, ModalBody, ModalFooter} from 'react-bootstrap';
 
-import ModalActions from 'app/actions/modalActions';
+
+import ModalActions, {ModalOptions} from 'app/actions/modalActions';
+
+export type RenderProps = {
```

Comment:
How I kind of feel like `ModalOptions` should just live here too heh

Reply:
@EvanPurkhiser sure, I can move it here